### ISSUE TITLE
Visible flickering of line chart tooltip

### ISSUE THREAD

**Issue by l2dy** [allegro/turnilo#633]
Commit dc3d7fb460c195ef9b68ca42951dc240b80a3e48 introduced the flickering if mouse moves from left to right, but not right-to-left.

Test Link: http://127.0.0.1:9090/#wiki/4/N4IgbglgzgrghgGwgLzgFwgewHYgFwhLYCmAtAMYAWcATmiADTjTxKoY4DKxaG2A5lHyh+NTDAAO3GhGJC8AM0RRiAXyYYAtsWQ5i+EAFE05APQBVACoBhRiAUQEaYjXkBtUGgCeE/QS36TDTECgYBdgEACi5YACbuoLEwNOhYuASRlgCMABJ2UM4S+KRZGj5+hOhy9KoAunVMUBJIaAkg3r4GBTICdrEQ2thQaQbkEN4AcnDa+Zh0wiDBoQRwsbHEsRHlXTFyfRDB5BzpIOtQ5MTY/b1MvjKYmwQg6oQD4/hZAAzqntv+A4FTgChiN/jMmKI4NgYAhaOMvAZMlkALKzeZ4UBLMIAradAj9QbDHD7Q7HAxwc6Xa78Z5MJCad54aEIBANEAqGR7PAeRYhFyXC7ktYbOwKOaadALDoVdZKGH0JhgRAwCq09p/EDaCnJfQ/XkKfnYQUrMD8AD6q3WmyYYpoEvoGPVeNOITg8rsSoQKoMaulBi1sGCtMxfOCRplxAQPBFNvFksdfvxrvdiuVqpeic1xG1Qb1S0NxpAcFNZvWUec1vscYdv2dsrdTg9aZ9GY1AZ1z1qtwg2BIsQAIsCiUN8G4QFQoSQEHZsNM5BI4IKmNAAEqYABGmHoXZAEh7fc4cwdReFm1UQA=

**Comment by adrianmroz** [comment]
You mean something like this: 
![tooltip](https://user-images.githubusercontent.com/14370131/83129426-180dde00-a0dd-11ea-9641-a2333ec6dda9.gif)

This is only case I could reproduce any weird behaviour. I agree that's not pretty, but it will take lower priority for now.

**Comment by l2dy** [comment]
Nope, there's no Y-axis shift in my case. I can reliably reproduce it with our data, but I can't share it here. I'll try to find a more reliable case to reproduce.

**Comment by adrianmroz** [comment]
y-axis shift is because sometimes third series jumps in and modal doesn't fit anymore. 

I could shift it a little bit, not by whole height. But that's different behaviour than x-axis, where tooltip should be glued to hovered line. Probably I should define two ways of fitting tooltip and let developer customize it per axis ...

**Comment by l2dy** [comment]
![Kapture 2020-10](https://user-images.githubusercontent.com/14329097/96707315-495bb300-13ca-11eb-834f-aed7e4d54f8a.gif)

@adrianmroz Here is a gif recording.
